lib/jasmine-reportportal-reporter.js

```diff
@@ -197,13 +245,62 @@ class ReportportalReporter {
 
         this.additionalCustomParams = {};
         this.finishParent();
+        this.startTime = null;
     }
 
     suiteDone() {
         let suiteDonePromise = this.client.finishTestItem(this.getParentId(), {});
 
         promiseErrorHandler(suiteDonePromise.promise);
         this.finishParent();
+        this.startTime = null;
+    }
+
+    makeWrapperAll(wrapped, funcStart, funcFinish) {
```

Comment:
May this function be moved to another file?

Reply:
Yes, I moved it to utils